tests/data/aws/kms.py

```diff
@@ -68,7 +68,7 @@
 
 DESCRIBE_GRANTS = [
     {
-        "KeyId": "9a1ad414-6e3b-47ce-8366-6b8f26ba467d",
+        "KeyId": "arn:aws:kms:eu-west-1:000000000000:key/9a1ad414-6e3b-47ce-8366-6b8f26ba467d",
```

Comment:
Does the API return a data shape that looks like this?

If not, I'm concerned that the tests will not reflect real conditions